packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -239,32 +240,60 @@ export default class PackMoveOffersCtrl {
       ) {
         if (option.name !== optionName) {
           set(option, 'selected', false);
+        } else {
+          set(option, 'selected', true);
         }
         if (optionName.match(/^gtr_\d{1,2}m_/) && option.selected === true) {
           optionComfort = true;
         }
         set(offer, 'gtrComfortActivated', optionComfort);
       }
+      if (optionName === 'none') {
+        set(option, 'selected', false);
+      }
     });
   }
 
-  updateOfferPriceAndGtr(value, offer, optionName) {
-    this.constructor.updateSelectedGtrOption(offer, optionName);
-    this.constructor.updateOfferDisplayedPrice(value, offer, optionName);
+  updateOfferPriceAndGtr(offer) {
+    this.constructor.updateSelectedGtrOption(offer, offer.gtrSelected);
+    this.constructor.updateOfferDisplayedPrice(1, offer, offer.gtrSelected);
   }
 
   static isChosen(option) {
     return option.choosedValue > 0;
   }
 
-  static isGtrOption(optionName) {
-    return optionName.startsWith('gtr_');
-  }
-
   static isGtrOptionExist(offer) {
     return offer.options.some((option) => option.name.startsWith('gtr_'));
   }
 
+  initGtrOptions() {
+    this.offers.forEach((offer) => {
+      // Initialize GTR options with a none value
+      let gtrOptions = [
+        {
+          label: this.$translate.instant('pack_move_no_gtr'),
+          name: 'none',
+        },
+      ];
+
+      Object.values(offer.options).forEach((opt) => {
+        if (opt.name.startsWith('gtr_')) {
+          const option = opt;
+          option.label = this.$translate.instant(`pack_move_${option.name}`, {
+            price: option.optionalPrice.text,
+          });
+          gtrOptions.push(option);
+        }
+      });
+
+      if (gtrOptions.length === 1) {
+        gtrOptions = [];
+      }
+      set(offer, 'gtrOptions', gtrOptions);
+    });
```

Comment:
**nitpick:** Please use `Array.prototype.map` for altering an array. In this snippet, each array item is overridden inside a `forEach` which makes it less readable.